EnglishFilter/sections/specific.txt

```diff
@@ -916,6 +916,9 @@ cyberciti.biz###custom_html-7
 ||souqsky.net/glx_*.js
 webvoyeur.com##.spot
 webvoyeur.com##.twocolumns > div.aside
+themoviesflix.co##.thecontent > center > div[style^="width:300px;height:250px;"]
+themoviesflix.co##div[id] > center div[style^="width: 320px; height: 100px;"]
```

Comment:
I don't see it `themoviesflix.co##div[id] > center div[style^="width: 320px; height: 100px;"]`